Neos.Neos/Documentation/GettingStarted/Installation.rst

```diff
@@ -16,9 +16,10 @@ Neos has at least the same system requirements as Flow. You can find them in the
 The most important requirements are:
 
 * A Webserver (Apache and Nginx are preferred but others work as well)
-* A Database (MySQL and MariaDB are preferred but any `Database supported by Doctrine DBAL
-  <http://www.doctrine-project.org/projects/dbal.html>`_ should work).
-* PHP >= 7.0.0 (make sure the PHP CLI has the same version)
+* A Database (MySQL > 5.7.7, MariaDB > 10.2.2 and PostgreSQL > 9.4 are preferred
+  but any `Database supported by Doctrine DBAL <http://www.doctrine-project.org/projects/dbal.html>`_
+  should work as well).
```

Comment:
Is that true ? as we don't provide migration for other DB engine ?

Reply:
It all boils down to creating your own migrations and/or using `doctrine:update` 🙈